web-ng/src/app/components/side-panel/side-panel.component.ts

```diff
@@ -14,36 +14,29 @@
  * limitations under the License.
  */
 
-import { Component, Input } from '@angular/core';
-import { LoadingState } from '../../services/loading-state.model';
-import { RouterService } from '../../services/router/router.service';
+import { Component } from '@angular/core';
+import {
+  RouterService,
+  SideNavContentMode,
+} from '../../services/router/router.service';
 import { ActivatedRoute } from '@angular/router';
-import { switchMap } from 'rxjs/operators';
-import { of, Observable } from 'rxjs';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'ground-side-panel',
   templateUrl: './side-panel.component.html',
   styleUrls: ['./side-panel.component.css'],
 })
 export class SidePanelComponent {
-  readonly sideNavContentType = SideNavContentType;
-  readonly sideNavContentType$: Observable<SideNavContentType>;
+  readonly sideNavContentMode = SideNavContentMode;
+  readonly sideNavContentMode$: Observable<SideNavContentMode>;
   readonly lang: string;
 
-  constructor(
-    private routerService: RouterService,
-    route: ActivatedRoute) {
+  constructor(private routerService: RouterService, route: ActivatedRoute) {
     routerService.init(route);
```

Comment:
`routerService` should be a singleton, and `init()` is already called in `MainPageComponent`, so should probably be removed.

Reply:
done